iidm/iidm-api/src/main/java/com/powsybl/iidm/network/VoltageAngleLimit.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public interface VoltageAngleLimit extends OperationalLimits {
+
+    public enum FlowDirection {
+        FROM_TO,
+        TO_FROM,
+        BOTH_DIRECTIONS
+    }
+
+    @Override
+    default LimitType getLimitType() {
+        return LimitType.VOLTAGE_ANGLE;
+    }
+
+    /**
```

Comment:
Can you add a name ?

Reply:
Yes I will do it. Name could be optional?

Reply:
Done. I considered it optional.